.github/workflows/release_candidate_validation.yml

```diff
@@ -77,3 +77,68 @@ jobs:
         if: ${{ env.SPARK_PROFILE >= 'spark3.3' }} # Only Spark 3.3 and above support Java 17
         run: |
           ./packaging/bundle-validation/ci_run.sh hudi_docker_java17 $HUDI_VERSION openjdk17 $STAGING_REPO_NUM
+
+  validate-release-maven-artifacts:
+    runs-on: ubuntu-latest
+    if: false
```

Comment:
I had created separate PR #12486 with this action turned on and Maven artifact validation succeeded - https://github.com/apache/hudi/actions/runs/12312424031/